addons/website/static/src/snippets/s_instagram_page/000.scss

```diff
@@ -0,0 +1,9 @@
+.s_instagram_page {
+    div.container-fluid {
+        padding-left: 0;
+        padding-right: 0;
+    }
```

Comment:
Is a container really needed if we disable its only purpose? Are we doing this in any other snippet?